### ISSUE TITLE
Revoir interface

### ISSUE THREAD

**Issue by barbogogo** [barbogogo/leedReader#27]
Revoir l'interface pour qu'elle soit plus ergonomique.

**Comment by cobalt74** [comment]
:D c'est le moment de mettre de l'orange ;)

**Comment by barbogogo** [comment]
@cobalt74 oui, puis si tu as des suggestions de présentation, n'hésite pas !
Le but n'est pas seulement de changer les couleurs mais de revoir complètement l'interface graphique et la navigation parmi les flux, articles, menu de configuratiln, ... ;-)

**Comment by cobalt74** [comment]
Quelques idées
- Accueil => tous les non lus
- Un menu caché a gauche ou droite qui cache la liste des flux et un lien vers la conf. (style application gmail)

**Comment by ghost** [comment]
Si tu pouvais donner un style inspiré de la version mobile de mon thème [HotBeer](https://github.com/misterair/HotBeer) je te serais éternellement reconnaissant ;-)

**Comment by barbogogo** [comment]
Le but est que l'application rappelle Leed.
Les premières captures d'écran sont disponibles : http://www.barbogogo.fr/premiere-vue-de-la-nouvelle-revision-de-leedreader/

Est-ce que tu peux un peu plus détailler la charte graphique de ton thème ?

**Comment by ghost** [comment]
La charte graphique c'est un noir texturé, un orange pétard (je te retrouverais le code hex), un gris #eee, et du blanc en fond pour l'article en cours de lecture.
Le principe est de mettre en avant les news et leur contenus sans être être distrait par le reste.

**Comment by barbogogo** [comment]
Ok, tu parles du noir texturé : https://github.com/misterair/HotBeer/blob/master/css/background.png ?
Le orange, je pense que tu as choisi celui-ci : http://www.colorhexa.com/fc4c03
L'application LeedReader n'ouvre qu'un article à la fois. Ensuite, il suffit de faire un swipe (Press, move, lift) vers la gauche ou la droite pour naviguer entre les articles.

**Comment by ghost** [comment]
tu peux te faire une idée en regardant mes screenshots: http://remitaines.com/hot-beer/
ma démo est en ce moment sur la version en dev de mon thème...

**Comment by barbogogo** [comment]
Merci pour ce lien.
Hier, j'ai installé ton thème et je n'avais ni vu comment mettre un article en favoris, ni comment mettre un article en non lu. Les icônes ne s'affichent pas si on ne les survole pas...

**Comment by ghost** [comment]
c'est normal que les icônes ne s'affichent qu'au survol, c'est pour épurer un maximum l'interface

**Comment by barbogogo** [comment]
ok, mais il faut connaitre... ;-)

**Comment by ghost** [comment]
Voilà globalement ce que j'imagine comme interface:
![leedreader](https://f.cloud.github.com/assets/4334834/855775/121bfb0a-f52f-11e2-810c-92ff7f97ecd8.png)

Pour le fond c'est [Celui-ci](https://github.com/misterair/HotBeer/blob/experimental/css/background.png) Celui de la branche master est trop clair...

**Comment by ghost** [comment]
Un bouton de synchronisation ce serait sympa pour ceux qui n'ont pas de cron

**Comment by barbogogo** [comment]
Bonjour, @cobalt74 @idleman @grimlokason 
[LeedReader v1.0 est prêt pour les tests](http://goo.gl/8hyhna).
Est-ce que vous pouvez prendre quelques minutes pour faire des tests et me donner vos retours ?
Merci pour votre collaboration dans le développement de cette application.
Barbogogo

**Comment by barbogogo** [comment]
@misterair ok, lje vais regarder ce qu'il est possible de faire sans trop toucher aux data échangées entre le smartphone et le serveur.

**Comment by grimlokason** [comment]
@barbogogo : L'interface est très bien. Il y a juste un "bug" qui peu être gênant, quand on est en portrait sur un article, qu'on pivote en paysage, on repasse à la liste des articles.
